src/app/common/components/SmartSearch/SmartSearch.tsx

```diff
@@ -152,81 +163,75 @@ const SmartSearch = (props: SmartSearchProps) => {
     }
   };
 
-  /**
-   * React on click on search icon addon in the query input.
-   */
-  const onSearchButtonClicked = () => {
-    setTimeout(() => {
-      redirectUserToSearchPage(query);
-    }, 10);
-  };
-
   /**
    * Rendering option of the search results
    * @param option
    */
   const onRenderOption = (option: any) => {
     return (
       <>
-        {option.type === 'VALIDATORS' && (
+        {option.type === 'Validators' && (
           <Link
             href={`${config.routes.VALIDATORS.HOME}/${option.public_key}`}
             className={classes.Link}
-            style={{ width: '100%' }}
+            style={{ width: '100%', marginBottom: 32 }}
           >
             <Grid item className={classes.BlackText}>
               0x{option.public_key}
             </Grid>
           </Link>
           )}
-        {option.type === 'OPERATORS' && (
+        {option.type === 'Operators' && (
           <Link
             href={`${config.routes.OPERATORS.HOME}/${option.id}`}
             className={classes.Link}
             style={{ width: '100%', marginBottom: 12 }}
           >
             <Grid container style={{ width: '100%' }} justify={'space-between'}>
               <Grid item container xs>
-                <Grid item className={classes.BlackText} style={{ marginRight: '5px' }}>
-                  {option.name}
+                <Grid item className={classes.SmartSearchOperatorDataOption}>
+                  <Grid item className={classes.BlackText} style={{ marginRight: '5px' }}>
+                    {option.name}
+                  </Grid>
+                  <Grid item className={classes.grayText}>
+                    ID: {option.id}
+                  </Grid>
                 </Grid>
                 <OperatorType type={option.operatorType} />
               </Grid>
-              <Grid item className={classes.BlackText}>
-                {/* {newLongStringShorten(option.address)} */}
-                ID: {option.id}
-              </Grid>
             </Grid>
           </Link>
           )}
       </>
     );
   };
 
+  const searchIconCondition = loading ? loading && !onFocus : !loading;
```

Comment:
condition can be simplified to `loading === onFocus`. Both loading and onFocus need to be the same for the condition to be false

Reply:
done